Coding_Buddy/projects/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from ..users.models import User
+from ..users.models import Skill
+
+class Project(models.Model):
+
+	users = models.ManyToManyField(User)
+
+	skills = models.ManyToManyField(Skill)
+
+	name = models.CharField(_("Name of the project"), null=True, blank=True, max_length=255)
```

Comment:
No to przydają się PR'y jak widać ;-)
zrób commita u siebie i go pushnij i powinno tu się zaktualizować.